help_to_heat/frontdoor/epc_api.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+import requests
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class EPCApi:
+    def __init__(self, token):
+        self.token = token
+
+    def get_access_token(self):
+        client_id = settings.EPC_API_CLIENT_ID
+        client_secret = settings.EPC_API_CLIENT_SECRET
+        token_url = "https://api.epb-staging.digital.communities.gov.uk/auth/oauth/token"
```

Comment:
Thanks for checking. I hadn't seen `os_api.py`.

I think for the interest of getting this ticket sorted, let's just leave `os_api.py` alone.

I'll note it down as a potential area for tech debt cleanup, but there's probably plenty of competing things we'd want to do in the codebase that would be higher priority!

Reply:
@jamiehumphries 
Thanks! I'm adding in new variables to both the `web` and `web.template` files (with placeholder values), so this will be the full list, anticipating for what we might need in future:

```
EPC_API_BASE_URL_STAGING="https://api.epb-staging.digital.communities.gov.uk"
EPC_API_BASE_URL="https://api.epb.digital.communities.gov.uk"
EPC_API_CLIENT_ID_STAGING="client_id"
EPC_API_CLIENT_ID="client_id"
EPC_API_CLIENT_SECRET_STAGING="client_secret"
EPC_API_CLIENT_SECRET="client_secret"
```